drools-ha/ha-core-infra/src/test/java/org/kie/hacep/LocalStorageKieSessionTest.java

```diff
@@ -127,7 +127,7 @@ public void deleteTest() throws ExecutionException, InterruptedException {
         assertEquals((Long) 0L, session.getFactCount().get());
     }
 
-    @Test(timeout = 20000)
+    @Test
```

Comment:
this timeout removal caused deadlock in previous reverted PR

Reply:
do we really think this will work at the end? 
WDYT @danielezonca @mariofusco ?